src/cowboy_session.erl

```diff
@@ -85,36 +85,41 @@ init([]) ->
 %% Internal functions
 %% ===================================================================
 
+get_cookie(#{ cookie := undefined } = Req) ->
```

Comment:
- Doesn't this give a bad-match whenever there is no `cookie` field in the `req()`?

- I realize the current implementation uses `cookie` as the name for this metadata, but I think we should stick to what the cowboy docs recommend (to prevent name clashes between different apps).

  _`cookie` also happens to be very general for something that should only be the value of the session cookie, and ignore all other cookies..._

  Perhaps we can use `'_cowboy_session_session'` as field instead?